frontend/packages/knative-plugin/src/components/overview/__tests__/EventPubSubResources.spec.tsx

```diff
@@ -11,64 +10,120 @@ import {
   EventTriggerObj,
 } from '../../../topology/__tests__/topology-knative-test-data';
 import EventPubSubResources, { PubSubResourceOverviewList } from '../EventPubSubResources';
+import PubSubSubscribers from '../EventPubSubSubscribers';
+import { Subscriber } from 'packages/knative-plugin/src/topology/knative-topology-utils';
 
 type EventPubSubResourcesProps = React.ComponentProps<typeof EventPubSubResources>;
 type PubSubResourceOverviewListProps = React.ComponentProps<typeof PubSubResourceOverviewList>;
-let itemData: OverviewItem & {
-  eventSources?: K8sResourceKind[];
-  eventingsubscription?: K8sResourceKind[];
-  connections?: K8sResourceKind[];
-};
+let sampleItemData: EventPubSubResourcesProps;
+
+const sampleSubscribers: Subscriber[] = [
+  {
+    name: 'subscriber',
+    kind: 'kind',
```

Comment:
shall we use proper kind in mock data here

Reply:
updated